1-js/02-first-steps/09-comparison/article.md

```diff
@@ -1,216 +1,209 @@
-# Comparisons
+# Համեմատություններ
 
-We know many comparison operators from maths.
+Մենք գիտենք համեմատության բազմաթիվ գործողություններ մաթեմատիկայից։
 
-In JavaScript they are written like this:
+JavaScript-ում դրանք գրվում են հետևյալ տեսքով․
 
-- Greater/less than: <code>a &gt; b</code>, <code>a &lt; b</code>.
-- Greater/less than or equals: <code>a &gt;= b</code>, <code>a &lt;= b</code>.
-- Equals: `a == b`, please note the double equality sign `==` means the equality test, while a single one `a = b` means an assignment.
-- Not equals. In maths the notation is <code>&ne;</code>, but in JavaScript it's written as <code>a != b</code>.
+- Մեծ/փոքր։ <code>a &gt; b</code>, <code>a &lt; b</code>։
+- Մեծ/փոքր կամ հավասար։ <code>a &gt;= b</code>, <code>a &lt;= b</code>։
+- Հավասար՝ `a == b`: Ուշադրություն դարձրեք, որ `==` գործողությունով ստուգվում է հավասարությունը, մինչ դեռ `a = b` նշանակում է վերագրում։
```

Comment:
`մինչ դեռ` -> `մինչդեռ`

Reply:
/done

Reply:
No need to left a `done` comment under each conversation. In posting that comment under a review/conversation is the wrong way of doing that. Sorry, I think I should have been more explicit. Just left a general comment under the issue (not conversation) saying `/done` when you're finished with your changes.